applications/GeoMechanicsApplication/custom_elements/geo_truss_element_base.cpp

```diff
@@ -206,34 +207,36 @@ void GeoTrussElementBase<TDim,TNumNodes>::
 
 //----------------------------------------------------------------------------------------
 template< unsigned int TDim, unsigned int TNumNodes >
-void GeoTrussElementBase<TDim,TNumNodes>::
+ void GeoTrussElementBase<TDim,TNumNodes>::
     CalculateConsistentMassMatrix(MatrixType& rMassMatrix,
                                   const ProcessInfo& rCurrentProcessInfo) const
 {
-    KRATOS_TRY;
+    KRATOS_TRY
+
+    static constexpr unsigned int NDof = TDim * TNumNodes;
+
     const double A = GetProperties()[CROSS_AREA];
     const double L = GeoStructuralMechanicsElementUtilities::CalculateReferenceLength3D2N(*this);
     const double rho = GetProperties()[DENSITY];
 
-    const unsigned int factor = double(TDim * TNumNodes);
+    const double factor = double(TDim * TNumNodes);
```

Comment:
```suggestion
    const double factor = TDim * TNumNodes;
```
I think the cast is automatically done in this case.